### PR TITLE
Fix/128031 kitchen sink refinements 2

### PR DESCRIPTION
**Pull request by paulmskim**
For ticket: https://central.tri.be/issues/128031

### CODE REVIEW

File: src/resources/postcss/base/forms/_sliders.pcss [hunk 1]
```diff
@@ -5,90 +5,103 @@
...
+		width: 120px;
+		vertical-align: middle;
 
-			/* -----------------------------------------------------------------------------
-			 * Track styles
-			 * ----------------------------------------------------------------------------- */
-			&::-webkit-slider-runnable-track {
-				@mixin form-control-slider-track;
-			}
+		/* -----------------------------------------------------------------------------
```
Comment (ryanurban) [inline src/resources/postcss/base/forms/_sliders.pcss]: All of these nested comment blocks look off on their tabbing for the 2nd & 3rd lines?

File: src/resources/postcss/base/forms/_tabs.pcss [hunk 2]
```diff
@@ -31,87 +65,87 @@
...
 		}
+	}
 
-		li {
+	.tribe-common-form-control-tabs__list-item {
 
-			&:first-child label {
-				@mixin form-control-select-list-item-first;
+		&:first-child label {
+			@mixin form-control-select-list-item-first;
```
Comment (ryanurban) [inline src/resources/postcss/base/forms/_tabs.pcss]: Noticing in general there are a lot of mixins for what potentially feels like one-off, non-reused styles around specific selectors. What and how are you deciding what to optimize for in how you're going about your use of mixins?

File: src/resources/postcss/base/forms/_text.pcss [hunk 3]
```diff
@@ -5,29 +5,34 @@
...
-		label {
-			@mixin visually-hide;
-		}
+	.tribe-common-form-control-text__label {
+		@mixin visually-hide;
+	}
 
-		input[type="text"] {
-			@mixin desktop-body-2-bold;
+	.tribe-common-form-control-text__input {
```
Comment (ryanurban) [inline src/resources/postcss/base/forms/_text.pcss]: Are we handling focus styles for form inputs?

Comment (paulmskim) [reply src/resources/postcss/base/forms/_text.pcss]: looks like not, will check with matt

File: src/resources/postcss/base/forms/_tabs.pcss (hunk 2 above)
Comment (paulmskim) [reply src/resources/postcss/base/forms/_tabs.pcss]: yeah, not sure what i was thinking with this (the mixin is just one style), will remove.

File: src/resources/postcss/base/forms/_sliders.pcss (hunk 1 above)
Comment (paulmskim) [reply src/resources/postcss/base/forms/_sliders.pcss]: yup, fixed!

File: src/resources/postcss/base/forms/_text.pcss (hunk 3 above)
Comment (paulmskim) [reply src/resources/postcss/base/forms/_text.pcss]: still in flux he says, will update once it comes out ;)

File: src/resources/postcss/base/forms/_tabs.pcss (hunk 2 above)
Comment (ryanurban) [reply src/resources/postcss/base/forms/_tabs.pcss]: Fwiw, this question isn't specific to just this instance of a mixin, curious about quite a few of the ones I see.

Comment (paulmskim) [reply src/resources/postcss/base/forms/_tabs.pcss]: Yup, I'll have another pass at them.

Comment (paulmskim) [reply src/resources/postcss/base/forms/_tabs.pcss]: So had a quick pass, there are a couple mixins that are only used once (a couple of the `-desktop` variations), which I'm sure could just be put into the class itself. But I figured when I did this that keeping all the styles in one place would be good, even if it's used only once.

**Pull request merged**